src/docs/project-maintainership.md

```diff
@@ -4,60 +4,26 @@ featured: ./images/featured/project-maintenance.png
 ---
 # Guidelines for Maintenance and Deprecation of GitHub Projects
```

Comment:
would you mind removing this? I noticed that adding a header at the top makes an awkward double-header at the top of the article.
![image](https://user-images.githubusercontent.com/7660110/166520187-6c1f21f8-46bc-4556-af70-4644281af522.png)